src/pages/NewChatSelectorPage.tsx

```diff
@@ -46,25 +32,17 @@ function NewChatSelectorPage(props) {
             >
                 <TopTab.Screen
                     name={CONST.TAB.NEW_CHAT}
-                    component={NewChatPage}
+                    component={NewChatPage as React.ComponentType}
                 />
                 <TopTab.Screen
                     name={CONST.TAB.NEW_ROOM}
-                    component={WorkspaceNewRoomPage}
+                    component={WorkspaceNewRoomPage as React.ComponentType}
```

Comment:
Let's wait for this to be merged first: https://github.com/Expensify/App/pull/34742

OR add `@ts-expect-error` here as described in TS guidelines
```suggestion
                    component={WorkspaceNewRoomPage}
```

Reply:
> Let's wait for this to be merged first: #34742
> 
> OR add `@ts-expect-error` here as described in TS guidelines

Yes, I think we should wait for it. There was an issue with "WorkspaceNewRoomPage".